presto-spark-base/src/test/java/com/facebook/presto/spark/PrestoSparkQueryRunner.java

```diff
@@ -369,7 +369,7 @@ public PrestoSparkQueryRunner(
         pluginManager.installPlugin(new HivePlugin("hive", Optional.of(metastore)));
 
         Map<String, String> properties = ImmutableMap.<String, String>builder()
-                .put("hive.experimental-optimized-partition-update-serialization-enabled", "true")
+                .put("hive.experimental-optimized-partition-update-serialization-enabled", "false")
```

Comment:
I think this is used for Java tests as well. Can we change buildSessionForTableWrite instead?

Reply:
@gggrace14 better to change https://github.com/prestodb/presto/blob/master/presto-native-execution/src/test/java/com/facebook/presto/spark/PrestoSparkNativeQueryRunnerUtils.java#L136 which only used by pos native to pass in the required configs.

Reply:
> @gggrace14 we can change https://github.com/prestodb/presto/blob/master/presto-native-execution/src/test/java/com/facebook/presto/spark/PrestoSparkNativeQueryRunnerUtils.java#L136 which only used by pos native to pass in the required configs.

Hi @miaoever , considered setting the session property in PrestoSparkNativeQueryRunnerUtils for native tests at first, but it calls PrestoSparkQueryRunner where the session property is set to true currently. Thus an error of duplicate property setting is raised.

I'm thus taking the approach that Masha suggests to set the session in the unit tests. Looks a more straightforward way.